assets/app/data/crowbar/services/upgrade.factory.spec.js

```diff
@@ -9,13 +9,43 @@ describe('Upgrade Factory', function () {
         mockedCreateAdminBackupResponse = {
             id: 42
         },
+        mockedNewDatabaseServerResponse = {
+            'database_setup': {
+                'success': true
+            },
+            'database_migration': {
+                'success': true
+            },
+            'schema_migration': {
+                'success': true
+            },
+            'crowbar_init': {
+                'success': true
+            }
+        },
+        mockedConnectDatabaseServerResponse = {
```

Comment:
Its not, but for the cost of one var we can visually separate them in the tests for easier comprehension of what it refers to.